compiler/damlc/lib/DA/Cli/Damlc.hs

```diff
@@ -840,7 +840,7 @@ execDocTest opts files = do
         -- This is horrible but we do not have a way to change the import paths in a running
         -- IdeState at the moment.
         pure $ nubOrd $ mapMaybe moduleImportPaths pmS
-    opts <- mkOptions opts { optImportPath = importPaths <> optImportPath opts}
+    opts <- mkOptions opts { optImportPath = importPaths <> optImportPath opts, optHaddock = Haddock True }
```

Comment:
Is that intended?

Reply:
Yes, we broke doctests when we switched the lexing mode to make HLint happy. We have tests but we only changed the options in the tests when they broke :man_facepalming: and not in doctest itself. I’ll look into adding an integration test that runs the actual executable on a failing doctest and verifies that it fails so this doesn’t happen again.

Reply:
Thanks for the explanation. Adding a test for this sounds good.

Reply:
Tests added